l10n/it-IT.xml

```diff
@@ -38,101 +38,101 @@
+                Value="Può %1 interrompersi?"/>
 <localized_string Key="AMS_MIDLETSUITEIMPL_PUSH_INTERRUPT_DIALOG_TITLE"
-                Value="Is it OK to Automatically Start?"/>
+                Value="Va bene avviare automaticamente?"/>
 <localized_string Key="AMS_MIDLETSUITEIMPL_PUSH_INTERRUPT_QUESTION"
-                Value="Information is arriving for %1. Is it OK for %1 to start up and receive this information? The currently running application will exit.&#010;&#010;Is it OK to start up?"/>
+                Value="Le informazioni stanno arrivando a %1. Va bene che %1 si avvii e ricevi queste informazioni? L'applicazione attualmente avviata si chiuderà.&#010;&#010;Va bene avviarla?"/>
 <localized_string Key="AMS_MIDLETSUITELDR_MIDLETSUITE_NOTFOUND"
-                Value="The MIDlet suite was not found."/>
+                Value="La suite di MIDlet non è stata trovata."/>
 <localized_string Key="AMS_MIDLETSUITELDR_PUSH_INTERRUPT_QUESTION"
-                Value="Information is arriving for %1.&#010;&#010;Is it OK for %1 to start up and receive this information?"/>
+                Value="Le informazioni stanno arrivando a %1.&#010;&#010;Va bene che %1 si avvii e ricevi queste informazioni?"/>
 <localized_string Key="AMS_MIDLETSUITELDR_CANT_LAUNCH_MISSING_CLASS"
-                Value="The application cannot be launched. One of the application classes appears to be missing. This could be due to a misnamed class. Contact the application provider to resolve the issue."/>
+                Value="L'applicazione non può essere avviata. Sembra che manchi una delle classi dell'applicazione. Questo può succedere a causa di una classe dal nome errato. Contattare il fornitore dell'applicazione per risolvere il problema."/>
 <localized_string Key="AMS_MIDLETSUITELDR_CANT_LAUNCH_ILL_OPERATION"
-                Value="The application cannot be launched. The application may have done an illegal operation. Contact the application provider to resolve the issue."/>
+                Value="L'applicazione non può essere avviata. L'applicazione potrebbe aver fatto un'azione illegale. Contattare il fornitore dell'applicazione per risolvere il problema."/>
 <localized_string Key="AMS_MIDLETSUITELDR_QUIT_OUT_OF_MEMORY"
-                Value="The application has unexpectedly quit because it ran out of memory."/>
+                Value="L'applicazione si è chiusa inaspettatamente poiché ha terminato la memoria."/>
 <localized_string Key="AMS_MIDLETSUITELDR_UNEXPECTEDLY_QUIT"
-                Value="The application has unexpectedly quit. Contact the application provider to resolve the issue."/>
+                Value="L'applicazione si è chiusa inaspettatamente. Contattare il fornitore dell'applicazione per risolvere il problema."/>
 <localized_string Key="AMS_MIDLETSUITELDR_RESOURCE_LIMIT_ERROR"
-                Value="No more concurrent applications allowed."/>
+                Value="Non sono più permesse applicazioni concorrenti."/>
```

Comment:
This should be "Non sono permesse altre applicazioni concorrenti", that is, you've reached the limit of maximum number of applications running concurrently.